stdlib/Markdown/src/render/terminal/render.jl

```diff
@@ -156,20 +183,31 @@ function terminline(io::IO, md::Image)
     terminline(io, "(Image: $(md.alt))")
 end
 
-terminline(io::IO, f::Footnote) = with_output_color(terminline, :bold, io, "[^$(f.id)]")
+function terminline(io::IO, f::Footnote)
+    print(io, styled"{markdown_footnote:[^$(f.id)]}")
+end
 
-function terminline(io::IO, md::Link)
-    url = !Base.startswith(md.url, "@ref") ? " ($(md.url))" : ""
-    text = terminline_string(io, md.text)
-    terminline(io, text, url)
+function terminline(io::AnnotIO, md::Link)
+    annots = if !isnothing(match(r"^(?:https?://|file://)", md.url))
```

Comment:
```suggestion
    annots = if occursin(r"^(https?|file)://", md.url)
```
as long as it is using a regex, might as well try to make it concise. I don't know if this necessarily is clearer to the average reader than 3 `startswith` calls, but it is readable enough to me

Reply:
I like this change :smiley:, thanks!